dev/TeachingTip/TeachingTip.xaml

```diff
@@ -618,17 +618,7 @@
                                             </StackPanel>
                                         </ScrollViewer>
                                         <Button x:Name="AlternateCloseButton"
-                                                Width="{StaticResource TeachingTipAlternateCloseButtonSize}"
-                                                Height="{StaticResource TeachingTipAlternateCloseButtonSize}"
-                                                Background="Transparent"
-                                                VerticalAlignment="Top"
-                                                HorizontalAlignment="Right"
-                                                BorderThickness="{ThemeResource TeachingTipAlternateCloseButtonBorderThickness}">
-                                            <Viewbox Width="{StaticResource TeachingTipAlternateCloseButtonGlyphSize}"
-                                                     Height="{StaticResource TeachingTipAlternateCloseButtonGlyphSize}">
-                                                <SymbolIcon Symbol="Cancel"/>
-                                            </Viewbox>
-                                        </Button>
+                                                        Style="{ThemeResource AlternateCloseButtonStyle}"/>
```

Comment:
nit: formatting
if it's just the button name and the style, it can all be put into one line

Reply:
fixed:)